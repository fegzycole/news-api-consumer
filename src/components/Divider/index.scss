@import "../../styles/palette";

.Divider {
  width: 100%;
  border: 1px solid $border-color;
}
