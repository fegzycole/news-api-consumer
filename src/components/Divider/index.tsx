import './index.scss';

const Divider = () => {
  return <div className="Divider" />;
};

export default Divider;
