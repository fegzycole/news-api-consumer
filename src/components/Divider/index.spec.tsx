import { render } from "@testing-library/react";

import Divider from "./index";

describe("<Divider />", () => {
  afterEach(() => {
    jest.clearAllMocks();
  });

  it("should match the Divider component snapshot", () => {
    const view = render(<Divider />);
    expect(view.container).toMatchSnapshot();
  });
});
