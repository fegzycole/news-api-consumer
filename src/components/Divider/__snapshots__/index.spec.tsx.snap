// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Divider /> should match the Divider component snapshot 1`] = `
<div>
  <div
    class="Divider"
  />
</div>
`;
